scripts/seed_db.py

```diff
@@ -0,0 +1,160 @@
+import csv
+import re
+import requests
+import json
+import time
+
+# Assumption: Given A1 -> N15
+#  A1 ............ N15            1  16 ........ 196
+#  A2              N2             2  17          197
+#  .               .              .  .           .
+#  .               .       --->   .  .           .
+#  .               .              .  .           .
+#  A15 ..........  N15            15 30 ........ 210
+
+
+# gdi_devpost -> csv is dumb lmao
+# moving -> list of projects that can move
+# notMoving -> list of projects that can't move
+# spots -> dict that goes table number:hack name
+# assignments -> list of spots that are taken
+gdi_devpost = "Does Your Hack Need To Stay At Your Current Table? " \
+              "(I.E. Hardware, Vr/Ar Hacks). If So, What Table" \
+              " Number Are You At?"
+moving, notMoving = {}, {}
+assignments = ["None | "] * 391
+
+
+class Project:
+    def __init__(self, project_url, attempted_challenges):
+        self.project_url = project_url
+        self.attempted_challenges = attempted_challenges
+        self.table_number = ""
+
+    def __str__(self):
+        return str(self.table_number) + " " + str(self.project_url)
+
+
+# table number to numeric value: A1 -> 1, N15 -> 210
+def table_to_number(table):
+    letter = table[0].upper()
+    num = table[1:]
+    return (ord(letter) - 65) * 15 + int(num)
+
+
+# numeric value to table number: 210 -> N15, 1 -> A1
+def number_to_table(number):
+    letter = chr((int(number) - 1)//15 + 65)
+    num = int(number) - (int(number) - 1)//15 * 15
+    return str(letter) + str(num)
+
+
+# checks if hacker responds with a table number, so they can't move
+def needs_to_stay(response):
+    return re.search('([a-zA-Z]+\d+)', response)
+
+
+# best domain name......
+def format_challenges(attempted_challenges):
+    if attempted_challenges is "":
+        return ""
+    output = []
+    challenges = attempted_challenges.split(',')
+    for challenge in challenges:
+        data = challenge.split(' - ')
+        prize = {
+            'company': data[1],
+            'challenge': data[0],
+            'winner': 'false'
+        }
+        output.append(prize)
+        json_output = json.dumps(output)
+    return json_output
+
+
+def already_in_db():
```

Comment:
Discussed offline - probably want to take this out since projects can have the same name. Also, DB should only be seeded _once_, and if a big update needs to happen then DB should be cleared beofre trying to seed with a new csv. Otherwise, small updates can be done from the admin panel.